.github/actions/deploy-artifact-docker/action.yml

```diff
@@ -114,7 +129,7 @@ runs:
       uses: docker/metadata-action@v5
       with:
         images: |
-          dotcms/dotcms
+          ${ inputs.image_name }
```

Comment:
You need double curly braces `{{` and `}}` here.

Reply:
Thanks yep missed this one. Fixed